pubnub/lib/src/core/keyset/keyset.dart

```diff
@@ -28,10 +37,19 @@ class Keyset {
 
   Keyset({
     required this.subscribeKey,
-    required this.uuid,
+    @deprecated this.uuid,
+    this.userId,
     this.publishKey,
     this.secretKey,
     this.authKey,
     this.cipherKey,
-  });
+  }) : assert(
```

Comment:
@are  This is one of the options for attempt making `one of the parameter is required`

TODO: Is there a way to introduce compilation warning!!